generate_qt_mappings.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
```

Comment:
Please leave this at `python`, and let's make sure the script is both Python2- and Python3-compatible.